src/tests/unit/tests/reports/components/report-sections/footer-text.test.tsx

```diff
@@ -1,19 +1,21 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { EnvironmentInfo } from 'common/environment-info-provider';
+import { EnvironmentInfoProvider } from 'common/environment-info-provider';
 import { shallow } from 'enzyme';
 import * as React from 'react';
 import { FooterText } from 'reports/components/report-sections/footer-text';
 
 describe('FooterText', () => {
     it('renders', () => {
-        const environmentInfo: EnvironmentInfo = {
-            extensionVersion: '1.2.3',
-            browserSpec: 'dummy browser version 1.0.1',
-            axeCoreVersion: '4.5.6',
-        };
+        const environmentInfoProvider = new EnvironmentInfoProvider(
```

Comment:
This should be a mock.
The main reason we were not using a mock before (with the `EnvironmentInfo`) is because there was not behavior, i.e. `EnvironmentInfo` is a simple type, a container of properties with not public/private methods.
By changing the dependency to be of type `EnvironmentInfoProvider` we are now using an actual class (even if we have an interface for it), at which point we should be using a mock, not the actual implementation. The reasoning here is: a unit test should isolate the test subject from all it's dependencies implementation details, in this case, the `EnvironmentInfoProvider` is such a dependency for the `FooterText`.